wechat.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+get_wechat_notifier_item() {
+  local notifier_items=$(
+    gdbus call --session \
+      --dest org.freedesktop.DBus --object-path /org/freedesktop/DBus \
+      --method org.freedesktop.DBus.ListNames |
```

Comment:
This won't work inside the sandbox, listnames is filtered

Reply:
I just remove the whole custom open & exit logic here. It looks like still working.